res/values/strings.xml

```diff
@@ -153,6 +153,9 @@
 <string name="p_ssid">SSID</string>
 <string name="p_ssid_summary">Station type (1..15; 9=Mobile, 10=APRS-IS)</string>
 <string name="p_ssid_entry">Enter the SSID for your station</string>
+<string name="p_units_title">Distance Units</string>
+<string name="p_units">Unit Preference</string>
```

Comment:
This preference should go into the "Display and Notifications" section. It can be the first item in the category. Then it should be named "Distance Units" and the sub-string should be removed, as it will be replaced by the selected value anyway.